cgi-bin/DW/Controller/Entry.pm

```diff
@@ -1147,7 +1164,23 @@ sub _do_edit {
         ditemid => $ditemid,
         editurl => $edit_url,
     );
-
+        
+    my $extradata = {
+        security => $form_req -> {security},
+        subject => $form_req -> {subject},
```

Comment:
Hmmm okay sort of nitpick, I think cleaner to have a temp variable `$security` instead of assigning (and reassigning) to `$extradata->security`. The reason being that we always know that `$extradata->{security}` will always just contain the ml strings.

Also, please escape the subject! You can either do that here, or apply the `| html` filter in the tempalte. I suggest the former.